src/applications/personalization/search-representative/config/basic-information/basicInformation.js

```diff
@@ -0,0 +1,34 @@
+export const schema = {
+  type: 'object',
+  title: 'What kind of representative are you looking for?',
+  properties: {
+    vso: {
+      type: 'boolean',
+    },
+    representativeOfVSO: {
+      type: 'boolean',
+    },
+    attorney: {
+      type: 'boolean',
+    },
+    claimsAgent: {
+      type: 'boolean',
+    },
+  },
+};
+
+export const uiSchema = {
+  'ui:description': 'Chech all that apply',
```

Comment:
[mockup](https://xd.adobe.com/view/b80b0931-826b-413a-9b07-1efff71cae16-ed58/screen/e4f0ce18-e81f-4ab4-99aa-22fdfa96dadc) uses the title `Check at least one`

Reply:
Yeah I took this wording from what we did on the 686c. But you are right, I updated this.